### ISSUE TITLE
Withdrawl from standard asset account to cash wallet not working

### ISSUE THREAD

**Issue by TMaddox** [firefly-iii/firefly-iii#2103]
I am running Firefly III version 4.7.11

**Description**
When I make a withdrawl and do not enter a expense account it is displayed that the money goes to my cash wallet (ATM-Withdrawl f.e.), but my cash wallet in asset accounts, does not receive any money. What am I doing wrong?

**Extra info**
OS: Ubuntu 18.04 LTS
nginx version: 1.15.8-1~xenial
php version: 7.2
I installed via composer

**Bonus points**

![grafik](https://user-images.githubusercontent.com/20703314/53049730-fd8b9480-3497-11e9-8739-7736746ed117.png)
![grafik](https://user-images.githubusercontent.com/20703314/53049876-522f0f80-3498-11e9-8fea-393f6cf7bc9d.png)

**Comment by JC5** [comment]
It's by design. If you want to transfer it to your cash wallet, use a transfer to do so. Otherwise it will be marked as cash and count as an expense.

**Comment by TMaddox** [comment]
Ah ok I think I get it, so if no expense account is added, it means "paid by cash to someone which is not frequently bought from", f.e. street food from a random seller, which one does not want to add as expense account?

**Comment by JC5** [comment]
Well to be honest I never track cash because it's impossible to keep a balanced book. Coins get lost, receipts are very rare and generally speaking it's messy.

So what I built is a system where an ATM withdrawals are expenses, no details saved. 

Some users however _do_ track their cash expenses which is why I added the option to create a "cash asset" account. But personally I don't use it, I don't track cash at all.
